cmd/kubeadm/app/phases/addons/addons_test.go

```diff
@@ -86,3 +92,71 @@ func TestCompileManifests(t *testing.T) {
 		}
 	}
 }
+
+// This unit test exists because tolerations weren't decoded correctly by kuberuntime.DecodeInto
```

Comment:
Sorry I can't remember what the bug was in 1.6. Was toleration recorded in annotation by then?